gradle.properties

```diff
@@ -1,3 +1,7 @@
 version=5.9.1
 nexusUser=
 nexusPassword=
+
+dockerJavaVersion=3.2.12
```

Comment:
I always wanted to define all external versions here but I feel convention is to store them in build.gradle:
https://github.com/ReactiveX/RxJava/blob/47189ac2ffb71304ca2e44c3d5f51eb9a0ea4e86/build.gradle#L15
https://github.com/web3j/web3j/blob/bfe7d12d7e37adad3955db9a8b28bf199dd0acb2/build.gradle#L12

Do you have many samples where gradle.properties is used instead?

Reply:
I decided to revert this change.

Dependency version definition is kind of an old-style practice.
It does not correspond to where the gradle project is going at the moment.

The answer to where do we define dependency versions is strongly related to how the source repositories are organised (multiple source repositories, all-in-one repository, ...).
This leads to a much broader discussion which may be a bit early in our situation.
I prefer to postpone it and to take the time to survey what is done in the community.
To complete the links you gave, I found those ones:
* https://github.com/mockito/mockito (dependencies applied from another file)
* https://github.com/spring-projects/spring-framework (dependency management in the root project of a gradle multi-project tree)
* https://github.com/open-telemetry/opentelemetry-java (dependencies defined in a plugin, its source code is located in the `buildSrc` folder)
* https://github.com/opensearch-project/OpenSearch (dependencies defined in a plugin, its source code is located in the `buildSrc` folder)

Reply:
Nice, thanks for these references, I have rarely seen this approach : https://github.com/mockito/mockito/blob/main/gradle/dependencies.gradle, always interesting.
I agree to come back to re factorization when internal architecture will change :+1: